docs/using/query-analytics.md

```diff
@@ -156,6 +156,16 @@ The *Explain* tab shows the `explain` output for the selected query, in Classic
 - MongoDB: JSON only.
 - PostgreSQL: Not supported.
 
+!!! note alert alert-primary "'Explain' for MongoDB"
+
+    To run Explain you need the same permissions as for executing the original query. For example, to run explain on `updates` you need update permissions.  
+
+Example: Grant the `explainRole` with update permissions.
```

Comment:
@percona-csalguero Is the example ("Example: Grant the..." part also meant to be included in the note?